### PR TITLE
duty cycle request -> master

### PR DESCRIPTION
**Pull request by maya1414**

### CODE REVIEW

File: src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java [hunk 1]
```diff
@@ -32,4 +29,8 @@ public static Phoenix6Request<Double> build(TorqueCurrentFOC torqueCurrentFOC) {
 		return new Phoenix6Request<>(torqueCurrentFOC.Output, torqueCurrentFOC, torqueCurrentFOC::withOutput);
 	}
 
+	public static Phoenix6Request<Double> build(DutyCycleOut dutyCycle) {
```
Comment (Nitay4) [inline src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java:32]:
```suggestion
	public static Phoenix6Request<Double> build(DutyCycleOut dutyCycleOut) {
```

**Review by Nitay4 — DISMISSED** [review]

**Review by YoniKiriaty — DISMISSED** [review]

**Review by Nitay4 — DISMISSED** [review]
why so skibidy

File: src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java [hunk 2]
```diff
@@ -1,9 +1,6 @@
 package frc.robot.hardware.phoenix6.request;
 
-import com.ctre.phoenix6.controls.PositionVoltage;
-import com.ctre.phoenix6.controls.TorqueCurrentFOC;
-import com.ctre.phoenix6.controls.VelocityVoltage;
-import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.controls.*;
```
Comment (YoniKiriaty) [inline src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java:3]: import all dont use *

File: src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java
Comment (drorta) [inline src/main/java/frc/robot/hardware/phoenix6/request/Phoenix6RequestBuilder.java:1]: I think you should also remove setPower everywhere its relevant, talk to @YHerm herman about that

**Review by drorta — CHANGES_REQUESTED** [review]

**Review by NoamRosenberg08 — DISMISSED** [review]

**Review by hillelv — APPROVED** [review]

File: src/main/java/frc/robot/hardware/phoenix6/motor/TalonFXMotor.java [hunk 3]
```diff
@@ -82,17 +82,6 @@ public void resetPosition(Rotation2d position) {
...
-	@Override
-	public void stop() {
-		motor.stopMotor();
-	}
-
-	@Override
-	public void setPower(double power) {
-		motor.set(power);
-	}
-
```
Comment (TheOnion313) [inline src/main/java/frc/robot/hardware/phoenix6/motor/TalonFXMotor.java:95]: This needs a replacement in the interface to ensure all motors support setting their power and stopping them

**Review by TheOnion313 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**